src/components/feedback-particle-effect/feedback-text-effects.ts

```diff
@@ -1,159 +1,85 @@
-import { TextParticle } from "@feedbackParticleEffect/text-particle";
-import { font } from "@common";
+import { hideShowElement, lang, Utils } from "@common";
+import { feedbackTextDefault, FONT_BASE_PATH } from "@constants";
+import { feedbackCustomFonts } from "@data/feedback-fonts";
 
 export class FeedbackTextEffects {
-  public context: CanvasRenderingContext2D;
-  public canvasWidth: number;
-  public canvasHeight: number;
-  public textX: number;
-  public textY: number;
-  public fontSize: number;
-  public lineHeight: number;
-  public maxTextWidth: number;
-  public particleDuration: number;
-  public startTime: number | null;
-  public particles: TextParticle[];
-  public gap: number;
-  public mouse: { radius: number; x: number; y: number };
-  public textWorker: Worker;
+  private feedbackTextElement: HTMLElement | null;
+  private hideTimeoutId: number | null = null;
```

Comment:
Question why is `null = null` why can't it be the same as in line 6?

Reply:
@bernhardccodev added this to the constructor instead. thank you